code.json

```
@@ -0,0 +1,74 @@
+[
+  {
+    "name": "ds-pipelines-targets-example-wqp",
+    "organization": "U.S. Geological Survey",
+    "description": "An example targets pipeline for pulling data from the Water Quality Portal (WQP)",
+    "version": "main",
+    "status": "Development",
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://github.com/USGS-R/ds-pipelines-targets-example-wqp/blob/main/LICENSE.md"
+        }
+      ]
+    },
+    "homepageURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp/",
+    "downloadURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp/archive/refs/heads/main.zip",
+    "disclaimerURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp/blob/main/DISCLAIMER.md",
+    "repositoryURL": "https://github.com/DOI-USGS/ds-pipelines-targets-example-wqp.git",
+    "vcs": "git",
+    "laborHours": 100,
+    "tags": [
+      "R",
+      "USGS"
+    ],
+    "languages": [
+      "R"
+    ],
+    "contact": {
+      "name": "Lauren Koenig-Snyder",
```

I use the following for all USGS science products so would prefer this edit:

```suggestion
      "name": "Lauren Koenig",
```